pkgs/applications/version-management/git-and-tools/git-big-picture/default.nix

```
@@ -1,23 +1,19 @@
-{ fetchFromGitHub, python3Packages, lib, stdenv, git, graphviz }:
+{ fetchPypi, python3Packages, lib, stdenv, git, graphviz, cram, pytest }:
```

fetchPypi can only be used in python packages like this.

```suggestion
{ python3Packages, lib, stdenv, git, graphviz, cram, pytest }:
```